src/components/Button/OutlineButton.tsx

```diff
@@ -53,10 +53,10 @@ export const OutlineButton: FC<OutlineButtonProps> = ({
 		>
 			<div
 				id='shadow'
-				className='absolute w-full h-full  bg-black z-0  bottom-0 animate-move-bounce-leave transform group-hover/OutlineButton:animate-move-bounce-enter'
+				className='absolute w-full h-full bg-black z-0 bottom-0 animate-move-bounce-leave transform group-hover/OutlineButton:animate-move-bounce-enter'
 			/>
```

Comment:
It was based on the animation that Mosaeedi provided. I suggest focusing on functionality and leaving the fancy stuff for the next phase. 😄